packages/integrations/lit/package.json

```diff
@@ -42,21 +42,21 @@
     "test": "mocha"
   },
   "dependencies": {
-    "@lit-labs/ssr": "^3.1.0",
-    "@lit-labs/ssr-dom-shim": "^1.1.0",
+    "@lit-labs/ssr": "^3.1.3",
+    "@lit-labs/ssr-dom-shim": "^1.1.1",
     "parse5": "^7.1.2"
   },
   "devDependencies": {
     "astro": "workspace:*",
     "astro-scripts": "workspace:*",
     "chai": "^4.3.6",
     "cheerio": "^1.0.0-rc.11",
-    "lit": "^2.7.0",
+    "lit": "^2.7.5",
     "mocha": "^9.2.2",
     "sass": "^1.52.2"
   },
   "peerDependencies": {
     "@webcomponents/template-shadowroot": "^0.2.1",
-    "lit": "^2.7.0"
+    "lit": "^2.7.5"
```

Comment:
Updating the peer deps would be a breaking change (and a major). Do we need to bump the `lit` version here?

Reply:
No, we don't nee to update this one, will downgrade.